diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/utils/KotlinParser.kt

```diff
@@ -27,6 +27,7 @@ import sun.reflect.ReflectionFactory
 /**
  * A class that wraps kotlin compiler's code parser and converts source code into AST
  */
+@Suppress("EMPTY_BLOCK_STRUCTURE_ERROR")
```

Comment:
Why? It should be fine with current snapshot

Reply:
I probably haven't updated my project.

Reply:
No, the project failed on this error